recipes-kernel/linux/linux-raspberrypi_4.19.bb

```diff
@@ -3,7 +3,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi:"
 LINUX_VERSION ?= "4.19.57"
 LINUX_RPI_BRANCH ?= "rpi-4.19.y"
 
-SRCREV = "55b89745a4782db50e86291bd1adee8fb44b84ad"
+SRCREV = "e95c533eba52d1879d94bdd1ede00548ad1cf622"
```

Comment:
Can you bump it to https://github.com/raspberrypi/linux/commit/246113692edbef9a438b31ab2dd0172a30ed5eb2 and also change `LINUX_VERSION to 4.19.58`

Reply:
Agreed. Pushed.